docs/content/doc/help/faq.en-us.md

```diff
@@ -0,0 +1,207 @@
+---
+date: "2016-11-08T16:00:00+02:00"
+title: "FAQ"
+slug: "faq"
+weight: 5
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "help"
+    name: "FAQ"
+    weight: 5
+    identifier: "faq"
+---
+
+# Frequently Asked Questions
+
+This page contains some common questions and answers.  
+Also see [Support Options]({{< relref "doc/help/seek-help.en-us.md" >}})
+
+* [How to migrate from Gogs/GitHub/etc. to Gitea](#how-to-migrate-from-gogs-github-etc-to-gitea)
+* [Where does Gitea store "x" file](#where-does-gitea-store-x-file)
+* [Not seeing a clone URL or the clone URL being incorrect](#not-seeing-a-clone-url-or-the-clone-url-being-incorrect)
+* [Custom Templates not loading or working incorrectly](#custom-templates-not-loading-or-working-incorrectly)
+* [Active user vs login prohibited user](#active-user-vs-login-prohibited-user)
+* [Setting up logging](#setting-up-logging)
+* [Adjusting your server for public/private use](#adjusting-your-server-for-public-private-use)
+  * [Preventing spammers](#preventing-spammers)
+  * [Only allow/block certain email domains](#only-allow-block-certain-email-domains)
+  * [Issue only users](#issue-only-users)
+* [Adding custom themes](#how-to-add-use-custom-themes)
+* [SSHD vs built-in SSH](#sshd-vs-built-in-ssh)
+* [Gitea is running slow](#why-is-gitea-running-slow)
+* [Translation is incorrect/how to add more translations](#translation-is-incorrect-how-to-add-more-translations)
+* [SSH Issues](#ssh-issues)
+* [Missing releases after migration repository with tags](#missing-releases-after-migrating-repository-with-tags)
+* [LFS Issues](#lfs-issues)
+
+
+## How to migrate from Gogs/GitHub/etc. to Gitea
+To migrate from Gogs to Gitea:
+
+* [Gogs version 0.9.146 or less]({{< relref "doc/upgrade/from-gogs.en-us.md" >}})
+* [Gogs version 0.11.46.0418](https://github.com/go-gitea/gitea/issues/4286)
+
+To migrate from GitHub to Gitea, you can use Gitea's [Migrator tool](https://gitea.com/gitea/migrator)
+
+To migrate from Gitlab to Gitea, you can use this non-affiliated tool:  
+https://github.com/loganinak/MigrateGitlabToGogs
+
+## Where does Gitea store "x" file
+* WorkPath
+  * Environment variable `GITEA_WORK_DIR`
+  * Else binary location
+* AppDataPath (default for database, indexers, etc.)
+  * `APP_DATA_PATH` from `app.ini`
+  * Else `%(WorkPath)/data`
+* CustomPath (custom templates)
+  * Environment variable `GITEA_CUSTOM`
+  * Else `%(WorkPath)/custom`
+* INI (config file)
+  * `-c` flag
+  * Else `%(CustomPath)/conf/app.ini`
+* SQLite Database 
+  * `PATH` in `database` section of `app.ini`
+  * Else `%(AppDataPath)/gitea.db`
+
+## Not seeing a clone URL or the clone URL being incorrect
```

Comment:
Not sure on best wording, but perhaps mention that the following settings can also control what clone links do/do not show up:
```
DISABLE_HTTP_GIT #if true no HTTP/HTTPS link
DISABLE_SSH #if true no SSH link
SSH_EXPOSE_ANONYMOUS #if false no SSH link for anon